repositories/repositories.bzl

```diff
@@ -1,27 +1,29 @@
 """Rules to load all dependencies of this project."""
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 
 # buildifier: disable=function-docstring
 def repositories():
-    excludes = native.existing_rules().keys()
+    versions = struct(
+        rules_python = "0.18.1",
```

Comment:
Can we jump straight to 0.27.1? That is the last version that supports Bazel 5. [Bazel 4 is already end-of-life](https://bazel.build/release#support-matrix), so I don't think we need to support it anymore.

Reply:
If you want that done in one big step with the other changes, i can do so

Reply:
Either way! We can always upgrade further in another PR